docs/Feature Documentation/Audio and Video Streaming Setup/index.md

```diff
@@ -1,66 +1,14 @@
 ## Initial Configuration
 
-### Install Packages
+### SDL Core Setup
 
-```
-sudo apt-get install git cmake build-essential libavahi-client-dev libsqlite3-dev chromium-browser libssl-dev libudev-dev libgtest-dev libbluetooth3 libbluetooth-dev bluez-tools gstreamer1.0* libpulse-dev
-```
-
-```
-sudo apt-get update
-sudo apt-get upgrade
-sudo ldconfig
-```
-
-### Clone the SDL Core Repository
-
-Clone the SDL Core [repository](https://github.com/smartdevicelink/sdl_core)
-
-```
-git clone https://github.com/smartdevicelink/sdl_core.git
-```
-
-CD into sdl_core/ and checkout the [master branch](https://github.com/smartdevicelink/sdl_core/tree/master)
-
-```
-git checkout origin/master
-```
-
-### Clone the SDL HMI Repository
-
-Clone the Web HMI [repository](https://github.com/smartdevicelink/sdl_hmi)
-
-```
-git clone https://github.com/smartdevicelink/sdl_hmi.git
-```
-
-CD into sdl_hmi/ and checkout the [master branch](https://github.com/smartdevicelink/sdl_hmi/tree/master)
-
-```
-git checkout origin/master
-```
-
-### Setup the Build Environment
-
-Create build folder outside of sdl_core/ directory
-
-```
-mkdir build
-cd build
-```
-
-Run CMAKE and install application
-```
-cmake ../sdl_core
-make
-make install
-```
+If you don't already have Core built, you can follow the [install and run guide](../../getting-started/install-and-run/).
```

Comment:
The install and run guide is only applicable to setting up sdl_core. If there is a guide for setting up the sdl_hmi we should link to that as well. If not, the `Clone the SDL HMI Repository` section should not be removed.

Reply:
added in https://github.com/smartdevicelink/sdl_core_guides/pull/71/commits/b07a1990e1799725a09996753b1db4bba73b664b